### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_NewJobSteps_Screenreader:Voice over is not announcing name of the button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5822]
[Check out Keros](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
GitHubTags: #A11yMAS; #A11yTCS; #MAS4.1.2; #MAS1.3.1; #A11y_AzureDataStudio; #MacOs; #DesktopApp
**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS:MacOS Mojave 10.14.5
**Additional Details:**
**Tool Name:Voice Over**
**MAS Reference: MAS1.3.1,MAS4.1.2**
**Repro Steps:**
1.Launch the Azure Data Studio application.
2.Enable the SQL server agent extension.
3.Naviagte to the SQL agent tab from Home page->Click on new job button->Select steps tab.
4.Go to the "New step" button hit enter/spacebar 
5.navigate to "Advanced' button and navigate with Tab till output file button.
5.Start Voice over
6.Observed that Voice over is not announcing the button name .
**Actual:**
Voice Over is not announcing button name and level associated with that button.
![image](https://user-images.githubusercontent.com/39204909/58797665-305c3180-861e-11e9-9847-12bcefb1d120.png)

**Expected:**
Voice Over should announce the name and level of the button so that user will understand the purpose of that button.
**User Impact:**
User with visual and motor disabilities will be confuse with the presence of unnamed button.
**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:**
MAS1.3.1,MAS4.1.2
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}
**Attachment for Reference:**
[Uploading 5822_AccessibilityA11y_AzureDataStudio_NewjobSteps_ScreenReader.pptx…]()
Does this issue occur when all extensions are disabled?: Yes
